src/style.css

```diff
@@ -0,0 +1,122 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.product__card {
+  display: flex;
+  flex-direction: column;
+  height: 408px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.product__img {
+  display: block;
+  height: 134px;
+  width: 160px;
+  padding: 32px 19px 40px;
+}
+
+.product__card-name {
+  margin: 0 16px 4px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product__card-code {
+  margin: 0 16px 16px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product__rating {
+  display: flex;
+  justify-content: space-between;
+  width: 166px;
+  margin: 0 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-top: 1px;
+  margin-right: 2px;
+  margin-left: 2px;
+}
+
+.stars :nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.product__comments {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  text-decoration: none;
+}
+
+.product__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  margin: 22px 16px 16px;
```

Comment:
add padding for the card, so you don't need to add horizontal margins to it's element children